test/integration/comms.cc

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+#include <chrono>
+#include <functional>
+#include <mutex>
+
+#include <ignition/msgs.hh>
+#include <ignition/transport/Node.hh>
+#include <ignition/utilities/ExtraTestMacros.hh>
+
+#include "ignition/gazebo/Server.hh"
+#include "ignition/gazebo/test_config.hh"  // NOLINT(build/include)
+
+#include "../helpers/EnvTestFixture.hh"
+
+using namespace ignition;
+using namespace gazebo;
+
+/////////////////////////////////////////////////
+class CommsTest : public InternalFixture<::testing::Test>
+{
+};
+
+/////////////////////////////////////////////////
+TEST_F(CommsTest, Comms)
+{
+  // Start server
+  ServerConfig serverConfig;
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+    "/examples/worlds/comms.sdf";
```

Comment:
```suggestion
  const auto sdfFile = ignition::common::joinPaths(std::string(PROJECT_SOURCE_PATH),
    "examples", "worlds", "comms.sdf");
```

Reply:
Changed in [beefb6b](https://github.com/ignitionrobotics/ign-gazebo/pull/1416/commits/beefb6bf950dc48ffb783c2c17ca2357ca8ad846).